Scripts/TagGenerator/TagGenerator.py

```diff
@@ -87,12 +93,14 @@ def get_cli_options():
                         help="Indicates this is expected to be the first tag.")
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="Enabled verbose script prints.")
+    parser.add_argument("-c", "--custom_tag", default=False,

```

Comment:
You have the default as a boolean, but then use the paramater as a string? Could we make this a bit more consistent? I wonder if using the custom tag directly as the tag value if not None or "" makes more sense then the boolean to indicate to use the major version. This would also avoid the oddity of trying to use a flag in the pipeline file.

Reply:
I agree the parameter type should at least be consistent.